lld/test/ELF/aarch64-got-relocations-pauth.s

```diff
@@ -0,0 +1,99 @@
+# REQUIRES: aarch64
+
+# RUN: rm -rf %t && split-file %s %t && cd %t
+
+# RUN: llvm-mc -filetype=obj -triple=aarch64 %p/Inputs/shared.s -o a.o
+# RUN: ld.lld -shared a.o -o a.so
+
+#--- ok.s
+
```

Comment:
unneeded blank line. we prefer not to add a blank line after `#---`

Reply:
Fixed in b277e3ba49797f368139c62f2f64b6be25af8679, thanks